setup/account_bank_statement_import_camt_oca/odoo/addons/account_bank_statement_import_camt_oca

```diff
@@ -0,0 +1 @@
+../../../../account_bank_statement_import_camt_oca
```

Comment:
@pedrobaeza is this needed?

Reply:
Well, it's a kind of trick/requirement for pip deployment (for which the company of this contributor, Acsone, is the main promoter). It's added by the bot at merge time in the same way as README is generated from fragments, but can be pre-generated directly in the PR.

Reply:
Maybe let's keep commits clean from autogenerated? For future migrators?

Reply:
The migration procedure rips-off this automatically, so no need to put it on a separate commit. I even prefer to be in the same.